packages/react-ui-kit/src/Form/Input.jsx

```diff
@@ -27,6 +27,15 @@ const placeholderStyle = css`
   text-transform: ${props => props.placeholderTextTransform};
 `;
 
+const dotRadius = 4;
+const dotSize = dotRadius + dotRadius;
```

Comment:
or this.. since you are avoiding magic numbers..

```js
const dotSize = dotRadius << 1;
```

Reply:
Maybe i will do that 😄 
But it wouldn't work with fractional numbers ...

Reply:
Jokes aside I prefer `<< 1` over `+` here :)

Reply:
also `dotDiameter` is more suitable in this case :)

Reply:
i changed everything, since svg allows me to scale the image as a whole